### PR TITLE
Add telemetry doc

### PR DESCRIPTION
**Pull request by eliecharra**
Related to https://github.com/cloudskiff/driftctl/issues/441 and https://github.com/cloudskiff/driftctl/issues/442

### CODE REVIEW

File: docs/usage/flags/disable-telemetry.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+id: disable-telemetry
+title: Disable telemetry
+---
+
+By default, driftctl checks send some telemetry data to help us improve the product.
```
Comment (sundowndev) [inline docs/usage/flags/disable-telemetry.mdx:6]: Not sure what you mean by "driftctl checks" ?

Comment (eliecharra) [reply docs/usage/flags/disable-telemetry.mdx]: Fixed

**Review by sundowndev — APPROVED** [review]

File: docs/usage/flags/disable-telemetry.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+id: disable-telemetry
+title: Disable telemetry
+---
+
+By default, driftctl send some telemetry data to help us improve the product.
```
Comment (wbeuil) [inline docs/usage/flags/disable-telemetry.mdx:6]: `driftctl sends telemetry data to ...`

File: docs/usage/flags/disable-telemetry.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+You can find below a sample of data.
```
Comment (wbeuil) [inline docs/usage/flags/disable-telemetry.mdx:7]: `You can find below a sample of what the data are.`

**Review by wbeuil — CHANGES_REQUESTED** [review]

**Review by wbeuil — APPROVED** [review]

**Pull request merged**